test/integration/patients/sidebar/action-sidebar.js

```diff
@@ -511,7 +512,7 @@ context('action sidebar', function() {
 
     cy
       .get('[data-activity-region]')
-      .should('contain', 'RoundingWell (System) added this Action')
+      .should('not.contain', 'RoundingWell (System) added this Action')
       .should('contain', 'Clinician McTester (Nurse) changed the Owner to Another Clinician')
       .should('contain', 'Clinician McTester (Nurse) changed the details of this Action')
       .should('contain', 'Clinician McTester (Nurse) changed the Due Date to ')
```

Comment:
you think we need a whole 'nother set of tests for a non-program action?

Reply:
was that the only test for `RoundingWell (System)` cause I don't want to lose that.  But yeah we probably should have another set of tests for the other option.. 1 for code coverage, but 2 `.should('not.contain', 'Whatever')` would also pass.. really we could probably add a test for an action with a program and have 2 events.. one create and the other the pathway one and then check for length of 1.

Reply:
alrighty, I added another round for 'created by program action'. take a look and lemme know